docs/topics/wildfly-swarm/docs/reference/fractions/hystrix.adoc

```diff
@@ -0,0 +1,120 @@
+= Hystrix
+
+
+.Maven Coordinates
+[source,xml]
+----
+<dependency>
+  <groupId>org.wildfly.swarm</groupId>
+  <artifactId>hystrix</artifactId>
+</dependency>
+----
+
+.Configuration
+
+swarm.hystrix.collapser.default.maxRequestsInBatch:: 
+The maximum number of requests allowed in a batch before this triggers a batch execution
+
+swarm.hystrix.collapser.default.requestCache.enabled:: 
+Indicates whether request caching is enabled for HystrixCollapser.execute() and HystrixCollapser.queue() invocations
```

Comment:
method invoking examples should be formatted. I lean towards backticks

Reply:
Note to self: not easily fixable in our sources.